concent_api/verifier/tasks.py

```diff
@@ -104,29 +110,68 @@ def blender_verification_order(
 
     unpack_archives(package_paths_to_downloaded_archive_names.values(), subtask_id)
 
-    frame_number = 1
-    render_image(frame_number, output_format, scene_file, subtask_id)
-
-    delete_source_files(package_paths_to_downloaded_archive_names[source_package_path])
+    result_files_list = get_files_list_from_archive(
+        generate_verifier_storage_file_path(package_paths_to_downloaded_archive_names[result_package_path])
+    )
 
-    blender_output_file_name = generate_full_blender_output_file_name(scene_file, frame_number, output_format.lower())
-    try_to_upload_blender_output_file(blender_output_file_name, output_format, subtask_id)
+    if len(frames) > len(result_files_list):
```

Comment:
this should be extracted to a separate function ;)

Reply:
Done